StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/NetworkingOTPView/NetworkingOTPView.swift

```diff
@@ -30,15 +30,26 @@ protocol NetworkingOTPViewDelegate: AnyObject {
 
 final class NetworkingOTPView: UIView {
 
+    enum TestModeValues {
+        static let otp = "000000"
+    }
+
     private let dataSource: NetworkingOTPDataSource
     weak var delegate: NetworkingOTPViewDelegate?
 
     private lazy var verticalStackView: UIStackView = {
-        let otpVerticalStackView = UIStackView(
-            arrangedSubviews: [
-                otpTextField,
-            ]
-        )
+        let otpVerticalStackView = UIStackView()
+
+        if dataSource.isTestMode {
+            let testModeBanner = TestModeAutofillBannerView(
+                context: .otp,
+                didTapAutofill: applyTestModeValue
```

Comment:
subjective but you could consider in-lining `applyTestModeValue` because its not referenced anywhere else

the thought pattern is that if a function is not used anywhere else but one place, then the simplest solution is to inline it - this theoretically reduces complexity because if one reads `NetworkingOTPView` they don't have to analyze whether `applyTestModeValue` is called multiple times in multiple places

Reply:
I know it's only used once, but I'm in favour of keeping this as is since in encapsulates a bit of the "business" logic away from the view logic. No strong opinion here though!